systemverilog-plugin/UhdmAst.cc

```diff
@@ -2486,6 +2486,17 @@ void UhdmAst::process_enum_typespec()
     const auto *enum_object = (const UHDM::enum_typespec *)handle->object;
     const auto *typespec = enum_object->Base_typespec();
 
+    if (current_node->str.empty()) {
+        // anonymous typespec, check if not already created
+        if (shared.anonymous_enums.find(enum_object) != shared.anonymous_enums.end()) {
+            // we already created typedef for this.
+            delete current_node;
+            current_node = new AST::AstNode(AST::AST_WIRETYPE);
```

Comment:
```suggestion
            current_node = make_node(AST::AST_WIRETYPE);
```
It sets file location info from obj_h which is nice when debugging.